docs/user/metal/install_upi.md

```diff
@@ -0,0 +1,107 @@
+# Install: User Provided Infrastructure
+
+The steps for performing a UPI-based install are outlined here. Several [Terraform][terraform-templates] templates are provided to help model your own.
+
+## Glossary
+
+1. Minimum compute requirements
+
+2. Network topology requirements
+
+3. DNS requirements
+
+4. Getting OS related assets for machines
+
+5. Getting Ignition configs for machines
+
+6. Watching your installation (bootstrap_complete, cluster available)
+
+## Compute
+
+The smallest OpenShift 4.x clusters require, 1 bootstrap machine, 3 control plane machines and at least 1 worker machine.
+
+NOTE: The bootstrap machine is required to bootstrap the OpenShift cluster on to the 3 control plane machines and can be decommissioned when the cluster has successfully bootstrapped.
+
+OpenShift 4.x clusters require the bootstrap and control plane machines to use the RHEL CoreOS [todo-link-to-rhcos]
+
+### Minimum Resource Requirements
+
+Processing
+Memory
+Storage
+Networking
+
+[todo-link-to-minimum-resource-requirements]
+
+## Network Topology Requirements
+
+OpenShift 4.x requires all nodes to have internet access to pull images for platform containers and phone home with telemetry data.
```

Comment:
nit: `home` is ?  maybe would be better to reword it such that the reason and the location is understood:

`phone home` -> `stream the telemetry data back to RedHat <telemetry central location>`